src/API/Util/Currencies.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace OnPay\API\Util;
+
+use OnPay\API\Exception\ApiException;
+
+class Currencies {
+    /**
```

Comment:
Indication of what ISO standard these currency codes are from, would be appreciated.